apps/src/code-studio/components/Grid.jsx

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
-import { SquareType } from '@cdo/apps/studio/constants';
+import { SquareType, WallTypeMask, WallCoordRowMask, WallCoordRowShift, WallCoordColMask, WallCoordColShift } from '@cdo/apps/studio/constants';
```

Comment:
nit: [our styleguide would prefer these be on separate lines](https://github.com/airbnb/javascript#modules--multiline-imports-over-newlines)